packages/graphql-mini-transforms/src/document.ts

```diff
@@ -33,12 +50,7 @@ export function cleanDocument(
     stripLoc(definition);
   }
 
-  // This ID is a hash of the full file contents that are part of the document,
-  // including other documents that are injected in, but excluding any unused
-  // fragments. This is useful for things like persisted queries.
-  const id = createHash('sha256')
-    .update(normalizedSource)
-    .digest('hex');
+  const id = generateId(print(document));
```

Comment:
I think we'd need this to continue referencing `normalizedSource` (which is printed + minified).

Reply:
The minifying is still done for the default implementation, it was just moved to `defaultGenerateId`. This allows the user-provided implementation of `generateId` to receive the non-minified version.

It was implemented this way as a result of testing my implementation with the signing. Since the server in that case is fed the queries by the client, and checks the signature to confirm they're legit, the query sent to the server needs to be identical to what was hashed and signed at build time. I found that the query which is sent to the server is the non-minified version, so if the signature was done on the minified version the signature wouldn't match the query payload. So, the unminified version of the query is provided to `generatedId`. It might be useful to export `minifySource` if a user may require it for implementing `generateId`, but I didn't think it necessary at the time.

Note this is from memory, and I haven't looked at this in months, so bear with me if I missed something.